modules/ocf_mirrors/manifests/monitoring.pp

```diff
@@ -27,7 +29,8 @@
     }
 
     cron { "${title}-health":
-      ensure => absent;
+      ensure => absent,
+      user   => mirrors;
```

Comment:
Nice, I'm not sure this ever worked before (lol), so thanks for changing that.

Reply:
Yeah, it would try and remove the service from root's crontab.

Reply:
lol nice